graphql_compiler/tests/test_helpers.py

```diff
@@ -204,6 +205,28 @@
 '''
 
 
+# A class holding all necessary backend-specific testing utilities.
+BackendTester = namedtuple('BackendTester', (
+    # Backend to be tested
+    'backend'
+
+    # Returns whether two emitted queries are the same, up to differences in syntax/whitespace
+    'compare_queries',
+
+    # An instance of backend.SchemaInfoClass consistend with the standard testing schema.
+    'schema_info',
+
+    # Given a SchemaInfo and a connection pool to a database, install the given schema into
+    # the database, erasing content if necessary.
+    'setup_schema'
+
+    # Given a SchemaInfo, a dict representation of data fitting the schema, and a connection pool
+    # to a database with the same schema, install the given data into the database, erasing any
+    # existing data.
+    'setup_data'
+))
```

Comment:
I will make the docstrings more rigorous when I implement the first `BackendTester`. I can't predict what's the best precise signature of each term here.